### PR TITLE
Make website REUSE compliant

### LINKED ISSUE #5: This repository is not REUSE-compliant
See title

From https://github.com/fsfe/reuse-tool/issues/29

### PR DESCRIPTION
**Pull request by mxmehl**
This PR makes the website repo REUSE compliant.

For some reason, `reuse-docs/practices/2.0.md` still throws an error, although I have clarified its actual license in the dep5 file. @carmenbianca, any idea why?

```
MISSING LICENSES

'BSD-2-Clause' found in:
  reuse-docs/practices/1.2.md

SUMMARY

Bad licenses: 0
Missing licenses: 1
Unused licenses: 0
Used licenses: AGPL-3.0-or-later, Apache-2.0, CC-BY-SA-4.0, CC0-1.0, GPL-3.0-or-later, MIT, OFL-1.1
Read errors: 0
Files with copyright information: 63 / 63
Files with license information: 63 / 63
```

Closes #5.

### CODE REVIEW

**Comment by carmenbianca** [comment]
I have a different output:

```
reuse._util - ERROR - Could not parse '[identifier]'
reuse.project - ERROR - site/content/faq.md holds an SPDX expression that cannot be parsed, skipping the file
reuse._util - ERROR - Could not parse '[identifier]'
reuse.project - ERROR - reuse-docs/faq.md holds an SPDX expression that cannot be parsed, skipping the file
NO LICENSE

The following files have no license(s):
  site/content/faq.md

NO COPYRIGHT

The following files have no copyright:
  site/content/faq.md

SUMMARY

Bad licenses: 0
Missing licenses: 0
Unused licenses: 0
Used licenses: AGPL-3.0-or-later, Apache-2.0, CC-BY-SA-4.0, CC0-1.0, GPL-3.0-or-later, MIT, OFL-1.1
Read errors: 0
Files with copyright information: 68 / 69
Files with license information: 68 / 69

Unfortunately, your project is not compliant with version 3.0 of the REUSE Specification :-(
```

Using the latest master branch of reuse-tool.

The error is understandable-ish, but annoying.

**Comment by mxmehl** [comment]
I wonder why you see faq.md, since the submodule reuse-docs is at tag v2.0?

I still don't understand it. I have `reuse-docs/*` in dep5, but `reuse lint` still complains about BSD-2-Clause being absent. Shouldn't it just silently believe what I am saying in dep5? What has precendence here?

File: site/themes/github-project-landing-page/static/css/landing-page.css [hunk 1]
```diff
@@ -1,11 +1,12 @@
-/*!
- * Start Bootstrap - Landing Page Bootstrap Theme (http://startbootstrap.com)
- * Code licensed under the Apache License v2.0.
- * For details, see http://www.apache.org/licenses/LICENSE-2.0.
- */
-
- body,
- html {
+/*
+ * SPDX-License-Identifier: MIT
```
Comment (carmenbianca) [inline site/themes/github-project-landing-page/static/css/landing-page.css]: Licensed under Apache, not MIT

**Comment by carmenbianca** [comment]
> I wonder why you see faq.md, since the submodule reuse-docs is at tag v2.0?

Oh! An artifact from previous mucking about on the website.

> I still don't understand it. I have reuse-docs/* in dep5, but reuse lint still complains about BSD-2-Clause being absent. Shouldn't it just silently believe what I am saying in dep5? What has precendence here?

I don't get your error :-( Technically neither has precedence, and both DEP5 *and* the file contents apply simultaneously. In a previous version of reuse-tool (i.e., 2017/2018) there was precedence, but this was removed.

**Comment by carmenbianca** [comment]
Ah, pardon me. I adjusted the submodule to `v2.0` and now I get the error. Because both DEP5 and file contents apply simultaneously, it expects BSD-2-Clause to be in the LICENSES/ directory.

This is obviously not desired behaviour, although technically "correct". This is a known issue: https://github.com/fsfe/reuse-tool/issues/29

**Comment by mxmehl** [comment]
> This is obviously not desired behaviour, although technically "correct". This is a known issue: fsfe/reuse-tool#29

Although I don't see the precendence bug mentioned in the issue after a quick read, I agree to the analysis that this is not the desired behaviour. We should allow for priorities, especially to handle false-positives like in this case (no matter if in a submodule or something else).

I am only uncertain whether we should either hardcode this (e.g. DEP5 > file) or make it configurable (in `.reuse` to make it consistent for all project users). I have a slight preference towards hardcoding it.

File: site/themes/github-project-landing-page/static/css/landing-page.css (hunk 1 above)
Comment (mxmehl) [reply site/themes/github-project-landing-page/static/css/landing-page.css]: Oh sorry. Thanks!

**Comment by carmenbianca** [comment]
The issue is that submodules in general are kind of broken. Instead of prioritising DEP5 or headers, being able to recursively lint submodules would fix the issue more cleanly.

**Comment by mxmehl** [comment]
OK. I will merge, but let's keep this quirk in mind to let the tool pass this repo 100% :)

**Pull request merged**